tests/testfiles/lunamark-markdown/no-auto-identifiers-with-attributes.test

```diff
@@ -0,0 +1,117 @@
+\def\markdownOptionAutoIdentifiers{false}
```

Comment:
I am in favor of removing both `no-auto-identifiers-with-attributes.test` and `no-gfm-auto-identifiers-with-attributes.test`.